services/tier-service/src/main/java/com/amazon/aws/partners/saasfactory/saasboost/model/Tier.java

```diff
@@ -83,6 +108,11 @@ public Builder description(String description) {
             return this;
         }
 
+        public Builder defaultTier(Boolean defaultTier) {
+            this.defaultTier = defaultTier;
+            return this;
+        }
+
         public Tier build() {
```

Comment:
The data access layer should be responsible for creating/maintaining the unique identifier for a persisted object not the domain model. We should implement this as we do in the rest of the SaaS Boost services were `id` is immutable and is owned by the persistence mechanism.